ui/webui/src/components/AnacondaAboutModal.jsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with This program; If not, see <http://www.gnu.org/licenses/>.
+ */
+import cockpit from "cockpit";
+
+import React, { useState, useEffect } from "react";
+import {
+    AboutModal,
+    Button,
+    Flex,
+    TextContent,
+    TextList,
+    TextListItem,
+} from "@patternfly/react-core";
+import { ExternalLinkAltIcon } from "@patternfly/react-icons";
+
+import { read_os_release as readOsRelease } from "os-release.js";
+import { getAnacondaVersion } from "../helpers/product";
+
+const _ = cockpit.gettext;
+
+const AboutModalVersions = ({ anacondaVersion }) => {
+    const arr = [["Anaconda", anacondaVersion], ["Blivet", "XXX"], ["Pykickstart", "XXX"], ["Python", "XXX"]];
+
+    return (
+        <>
+            <TextContent id="subtitle">
+                <h2>{_("Powered by Anaconda")}</h2>
+            </TextContent>
+            <TextContent />
+            <TextContent>
+                <TextList id="about-modal-content" component="dl">
+                    {arr.map((item) => (
+                        <React.Fragment key={item[0]}>
+                            <TextListItem component="dt">{item[0]}</TextListItem>
+                            <TextListItem component="dd">{item[1]}</TextListItem>
+                        </React.Fragment>
+                    ))}
+                </TextList>
+            </TextContent>
+        </>
+    );
+};
+
+export const AnacondaAboutModal = ({ isModalOpen, setIsAboutModalOpen }) => {
+    const toggleModal = () => {
+        setIsAboutModalOpen(!isModalOpen);
+    };
+
+    const [anacondaVersion, setAnacondaVersion] = useState("");
+    const [prettyName, setPrettyName] = useState("");
+    useEffect(() => {
+        getAnacondaVersion().then(content => setAnacondaVersion(content));
+        readOsRelease().then(osRelease => setPrettyName(osRelease.PRETTY_NAME));
+    }, []);
+
+    return (
+        <>
+            <AboutModal
+              isOpen={isModalOpen}
+              onClose={toggleModal}
+              trademark="Anaconda © 2023 Red Hat, Inc."
+              productName={prettyName}
```

Comment:
I would say that according to the design the 'Power by anaconda' string is part of the title and not the body.  then you can remove all the custom CSS for the marins on the subtitle. 


I would expect something like:
``` diff
+                  <Stack hasGutter>
+                      <StackItem>{prettyName}</StackItem>
+                      <StackItem className="subtitle">{_("Powered by Anaconda")}</StackItem>
+                  </Stack>

```